packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -79,6 +79,9 @@ export class HorizontalBarChart extends FASTElement {
     // Iterate through all chart points and populate the map
     this.data.forEach(dataSeries => {
       dataSeries.chartData!.forEach(point => {
+        if (point.placeholder) {
```

Comment:
what is the color associated with placeholder bar

Reply:
Passed in the data